src/app/ngrx/effects/cms.effects.ts

```diff
@@ -71,17 +105,17 @@ export class CmsEffects {
     });
   }
 
-  getEpisodePodcastDistribution(episode: EpisodeModel): Observable<{}> {
-    return episode.doc.followItems('prx:distributions').map((docs: HalDoc[]) => {
-      return docs.filter((doc => doc['kind'] === 'episode' && doc['url']))
-        .map((distro: HalDoc) => {
-          episode.feederUrl = distro['url'];
-          const urlParts = episode.feederUrl.split('/');
-          if (urlParts.length > 1) {
-            episode.guid = urlParts[urlParts.length - 1];
-          }
-        });
-    });
+  getEpisodeColor(episodeIndex: number): string {
+    return getColor(EPISODE_PAGE_SIZE, episodeIndex);
+  }
+
+  // if none of the incoming episodes are already on the route, we want to add the first 5
+  chartIncomingEpisodes(episodes: EpisodeModel[]) {
```

Comment:
Ended up refactoring this somewhat, as it's called from a different point in the effect now.